preview/Win7Msix/Win7MSIXInstaller/InstallComplete.hpp

```diff
@@ -6,14 +6,15 @@
 class InstallComplete : IPackageHandler
 {
 public:
-	HRESULT ExecuteForAddRequest();
+    /// Send install complete message to UI if cancel button was not clicked, else uninstalls the app

```

Comment:
> if cancel button was not clicked, else uninstalls the app [](start = 43, length = 58)

you're not uninstalling here, and i think at a certain point (this point) you reach the point of no return and can no longer cancel anyway, so i don't think you should mention anything about cancel.

Reply:
Updated comment.